doc/getting_started.rst

```diff
@@ -0,0 +1,159 @@
+The ``pyGeo`` package manages all geometric parameterization in terms of point sets.
+``pyGeo`` point sets are always ``numpy`` arrays of dimension :math:`N_{\text{points}} \times 3`.
+
+Shape optimization begins with a baseline geometry.
+To keep things simple, we'll start with a cylinder centered on the point (0.5, 0.0, 0.0) extending 1.0 units along the z axis, with a radius of 0.5.
+The image below shows a simple pointset on the edge of the cylinder, along with a cylinder surface for reference.
+
+.. image:: ../examples/ffd_cylinder/images/cylinder_only_3d.png
+   :width: 450
+   :align: center
+
+This pointset was generated using the following code snippet:
+
+.. literalinclude:: ../examples/ffd_cylinder/runFFDExample.py
+    :end-before: # rst create DVGeo
+
+----------------------
+Creating an FFD volume
+----------------------
+
+We'll use the free form deformation (FFD) approach to parameterize the geometry by creating a ``pygeo.DVGeometry`` object.
+
+The FFD method details are fully described in `the paper <http://doi.org/10.2514/6.2010-9231>`_, but for now it suffices to understand FFD qualitatively.
+First, the user creates an FFD *volume*, defined by a structured grid of *control points*.
+The FFD volume can be of arbitrary shape, but they tend to be rectangular or prismatic in form, and they must completely enclose the point set geometries to be added.
+Next, a point set is *embedded* in the FFD volume.
+Finally, the control points defining the FFD volume can be moved in space.
+
+As the control points move, they stretch and twist the FFD volume as if it was a block of Jello.
+The points embedded in the volume will also strech and move in a consistent way.
+The image below shows the cylinder we made embedded in a cube-shaped FFD volume. The FFD control points are depicted with the blue dots.
+
+.. image:: ../examples/ffd_cylinder/images/cylinder_ffd_3d.png
+   :width: 450
+   :align: center
+
+``pyGeo`` expects the FFD volume to defined in the Plot3D file format.
+Points must be defined in a complete, ordered 3D grid.
+The structured grid axes are typically referred to as i, j, and k axes, since they do not necessarily align with the x, y, and z spatial axes.
+Each dimension must be of length at least 2.
+
+For 3D aerodynamic shape problems where the pointset is a surface, we usually leave one dimension of length 2 and it serves as the primary perturbation direction (e.g. up and down for an airfoil design problem).
+The other two directions (generally streamwise and spanwise) should have more points.
+Airfoil design problems may have just two points in the spanwise direction (as pictured here).
+
+The following script creates the DVGeometry object and generates the pictured cube-shaped FFD volume.
+Depending on the user's skill it may be possible to create FFD volumes which conform more closely to the pointset.
```

Comment:
Should we link this to the fitted FFD function from #139?

Reply:
yeah we should rephrase it and add the link to that function